packages/gen-ai/frontend/src/app/Chatbot/ChatbotMain.tsx

```diff
@@ -104,6 +124,30 @@ const ChatbotMain: React.FunctionComponent = () => {
         />
       }
       loadError={lsdStatusError || modelsError}
+      headerAction={
+        lsdStatus &&
```

Comment:
I will need to create a hook to check more specifically about the lsd status. But this is fine here is in PR.